apps/heft/src/utilities/TaskPackageResolver.ts

```diff
@@ -206,9 +234,7 @@ export class TaskPackageResolver {
     }
 
     // Look for the governing package of "tsconfigPath"
-    const rigPackagePath: string | undefined = TaskPackageResolver._packageJsonLookup.tryGetPackageFolderFor(
-      tsconfigPath
-    );
+    const rigPackagePath: string | undefined = this._packageJsonLookup.tryGetPackageFolderFor(tsconfigPath);
```

Comment:
Obligatory 'more async'

Reply:
There isn't an async variant of `tryGetPackageFolderFor` yet.